index.html

```diff
@@ -61,8 +61,231 @@ <h1 class="box1">
+                     <ul class="project_job_title_info_container">
+                         <li class="job_title_info">
+                           Back End Dev
+                         </li>
+                         <li class="job_title_info">
+                           2015
+                         </li>
+                     </ul>
+                   </div>
+                   <p class="project_full_desc">
+                     A daily selection of privately personalized reads; no accounts or sign-ups required.
+                   </p>
+                   <div class="project_tags_container">
+                       <p class="project_tag">html</p>
+                       <p class="project_tag">css</p>
+                       <p class="project_tag">javaScript</p>
+                   </div>
+                   <button type="button" class="view_project_btn">
+                       See Project
+                   </button>
+                </div>
+              </li>
+              <li class="project">
+                <img src="images/project4.png" alt="Project Image 4">
+                <div class="project_description">
+                   <h2 class="project_title">
+                    Multi-Post Stories
+                   </h2>
+                   <div class="project_info">
+                     <p class="project_title_II">
+                      CANOPY
+                     </p>
+                     <ul class="project_job_title_info_container">
+                         <li class="job_title_info">
+                           Back End Dev
+                         </li>
+                         <li class="job_title_info">
+                           2015
+                         </li>
+                     </ul>
+                   </div>
+                   <p class="project_full_desc">
+                     A daily selection of privately personalized reads; no accounts or sign-ups required.
+                   </p>
+                   <div class="project_tags_container">
+                       <p class="project_tag">html</p>
+                       <p class="project_tag">css</p>
+                       <p class="project_tag">javaScript</p>
+                   </div>
+                   <button type="button" class="view_project_btn">
+                       See Project
+                   </button>
+                </div>
+              </li>
+             </ul>
+            </section>
 
+            <section class="about_container">
+              <h1 class="box1">
+                  About Myself
+              </h1>
+              <p class="box2">
+                Hello I'm a software developer! I can help you build a product, 
+                feature or website Look through some of my work and experience! 
+                If you like what you see and have a project you need coded, don't hestiate to contact me.
+              </p>
+              <div class="box3">
+                  LET'S CONNECT
+              </div>
```

Comment:
- [ ] Let's consider using a header tag `hx` for this title instead of a `div` tag where `hx` can be h1, h2, h3... etc